### DISCUSSION TITLE
DAI 2023-2024 - Practical work 4 - Bleuer Rémy, Graf Calvin, Lopez Esteban, Sottile Alan

### DISCUSSION THREAD

**Discussion by AlanSottile** [heig-vd-dai-course/heig-vd-dai-course#184] [Show and tell]
Nous souhaitons réaliser une application qui nous permettera d'ajouter des jeux videos, qu'on va lier à un score et une durée de jeu.

Fonctionnalité :
- Ajouter / modifier / supprimer un utilisateur
- Un utilisateur peut ajouter / modifier / supprimer un jeu de sa liste qui va contenir un score et une durée de jeu
- Récupérer un classement des meilleurs utilisateur par score pour un jeu
- Récupérer la liste de tous les jeux ajoutées par tous les utilisateurs
- Récupérer la liste de jeu d'un utilisateur en fonction de son pseudo

**Comment by RemyBlr** [comment]
Here's our repository :
https://github.com/RemyBlr/dai-pw-4-bleuer-graf-lopez-sottile

**Comment by RemyBlr** [comment]
Our latest commit [4fb20eb](https://github.com/RemyBlr/dai-pw-4-bleuer-graf-lopez-sottile/commit/4fb20eb98b0ce35a7c5d3082372fcb72130e7fb5)
